games/l.yaml

```diff
@@ -79,78 +79,6 @@
   type: remake
   updated: 2017-10-15
 
-- name: The Legend Of Zelda - Navi's Quest
```

Comment:
we want to ignore common words like "The" when ordering
https://en.wikipedia.org/wiki/Alphabetical_order#The_and_other_common_words
See https://github.com/opengaming/osgameclones/blob/master/_ext.py#L216-L223 for an example of how to handle this
This is so we don't end up with a ton of entries in `t.yaml` for example

Reply:
Good idea. I will implement it in a further version. This will move entries currently in t.yaml like The Battle for Wesnoth somewhere else.